.github/bazel_test.yml

```diff
@@ -0,0 +1,11 @@
+name: Bazel Test
```

Comment:
Does this file need to be in .github/workflows/bazel_test.yml to trigger?

Reply:
Oops, forgot to move to workflow/